includes/class-used-css.php

```
@@ -0,0 +1,1206 @@
+			if ( ! $fs->is_dir( $root ) ) {
+				return true;
+			}
+
+			$success = true;
+			$dirs    = $fs->dirlist( $root );
+			if ( ! $dirs ) {
+				return true;
+			}
+
+			// Use a recursive iterator to find all used-css.css files at any depth.
+			$iterator = new \RecursiveIteratorIterator(
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '850', column '20'). (undefined)

(MissingImport)